Index: src/weka/classifiers/rules/odri/Pair.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package weka.classifiers.rules.odri;\n\n/**\n * Created by suhel on 23/03/16.\n */\n\n/**\n * Helper class\n *\n * @param <K> key\n * @param <V> value\n */\npublic class Pair<K, V> {\n  final public K key;\n  final public V value;\n\n  public Pair(K key, V value) {\n    this.key = key;\n    this.value = value;\n  }\n}\n
===================================================================
diff --git a/src/weka/classifiers/rules/odri/Pair.java b/src/weka/classifiers/rules/odri/Pair.java
--- a/src/weka/classifiers/rules/odri/Pair.java	
+++ b/src/weka/classifiers/rules/odri/Pair.java	
@@ -11,6 +11,7 @@
  * @param <V> value
  */
 public class Pair<K, V> {
+  final public static Pair NONE = new Pair(null, null);
   final public K key;
   final public V value;
 
